verilog/tools/syntax/verilog_syntax.cc

```diff
@@ -298,8 +298,11 @@ int main(int argc, char** argv) {
       continue;
     }
 
-    // TODO(hzeller): make preprocessing configurable with flags.
-    const verilog::VerilogPreprocess::Config preprocess_config;
+    // TODO(hzeller): is there ever a situation in which we do not want
```

Comment:
If you are fed an input which is known to be preprocessed, you might want to disable this (flag).
C compilers have an option like this to skip preprocessing.

Reply:
But even then, there would be no difference.